### DISCUSSION TITLE
HTML 네이밍 컨벤션 관련 건의

### DISCUSSION THREAD

**Discussion by LeeMooho** [NC4Mini/NC4_mini_project-1.10-#31] [Announcements]
HTML 네이밍 컨벤션이 혼용, 오용되고 있는 것 같아서 내용공유드립니다.
제가 알아본 바로는 카멜케이스보다 케밥 케이스 혹은 스네이크 케이스가 보편적인 것 같아서 내용 공유드립니다.

layoutSidebar.html (X)
layout-sidebar.html (O) 케밥 케이스
layout_sidebar.html (O) 스네이크 케이스

이미 CSS는 스네이크 케이스로 작성되어 있어서 변경하기엔 너무 많은 시간이 소요될 것 같습니다. 
html 앞단 파일명만 통일하면 어떨까요? 

저는 CSS와 동일하게 스네이크 케이스로 작성되면 어떨까합니다.

파일명을 바꾸기로 결정되면 본인이 작성한 파일만 변경해서 깃에 올리는 것으로 하겠습니다.

**Comment by Robbie-Kim** [comment]
### 01.19 적용완료
- cart디렉토리 5개 templates 완료
- item 디렉토리 2개 완료

**Comment by LeeMooho** [comment]
layouts 적용완료
